src/components/nav-header-panels/user-menu-panel.component.tsx

```diff
@@ -31,7 +42,24 @@ const UserMenuPanel: React.FC<UserMenuPanelProps> = ({
         allowedLocales={allowedLocales}
         open={expanded}
       ></UserMenu>
-      <Switcher aria-label="Switcher Container">
+      <Switcher className={styles.switcher} aria-label="Switcher Container">
+        <UserAvatarFilledAlt20 />
+        <p>{user.person.display}</p>
+      </Switcher>
+      <Switcher className={styles.switcher} aria-label="Switcher Container">
+        <ExtensionSlot
```

Comment:
Can we think a little bit beyond this one use case?

Today, we add the location picker under user profile. Tomorrow, we add some user information. Then in a week, we'll add some audits for the user or anything else. We should not require additional extension slots. One slot should be sufficient...

Naming the slot "location-changer" does not make much sense. I would rather encourage to use names that describe the location - not what we *now* think will enter at this location.

In this case I would expand the usage (including the `Switcher`) and call it "user-panel" or similar. Generally, the idea is to have this part extensible such that new functionality can be provided by other microfrontends in the future.